### PR TITLE
Update Venmo Package Name

### PR DESCRIPTION
**Pull request by tdchow**
### Summary of changes

 - Update venmo package name to `com.braintreepayments.api.venmo`
 - Bump browser switch SDK to `3.0.0-beta2-SNAPSHOT`
 
 This Browser Switch PR needs to be merged before CI passes: https://github.com/braintree/browser-switch-android/pull/99

### Checklist

 - [x] Added a changelog entry
 - [ ] Relevant test coverage

### CODE REVIEW

File: Venmo/src/test/java/com/braintreepayments/api/venmo/VenmoRequestUnitTest.java [hunk 1]
```diff
@@ -1,10 +1,14 @@
-package com.braintreepayments.api;
+package com.braintreepayments.api.venmo;
 
 import static junit.framework.TestCase.assertTrue;
 import static org.junit.Assert.assertEquals;
 
 import android.os.Parcel;
 
+import com.braintreepayments.api.venmo.VenmoLineItem;
```
Comment (saperi22) [inline Venmo/src/test/java/com/braintreepayments/api/venmo/VenmoRequestUnitTest.java:8]: these shouldn't be required

**Review by saperi22 — APPROVED** [review]
some cleanup is required - fixing linter will probably fix it all.

File: CHANGELOG.md [hunk 2]
```diff
@@ -42,6 +42,7 @@
     * Add `VenmoClient#createPaymentAuthRequest`
     * Move `showVenmoInGooglePlayStore` to `VenmoLauncher`
     * Remove `isVenmoAppSwitchAvailable` and `isReadyToPay` (no longer required as Venmo flow will fallback to web if app is not installed)
+    * Update package name to `com.braintreepayments.api.venmo`
```
Comment (saperi22) [inline CHANGELOG.md:45]: should this go to the top of the list?

Comment (tdchow) [reply CHANGELOG.md:45]: I've been adding all of the package name updates to the bottom of each section. Should we surface them to the top?

**Review by jaxdesmarais — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (tdchow) [reply CHANGELOG.md]: I moved all of the package rename bullets to the top of each section 👍

**Pull request merged**